youtube_transcript_api/_api.py

```diff
@@ -83,10 +85,13 @@ def get_transcript(video_id, languages=None):
         :type languages: [str]
         :return: a list of dictionaries containing the 'text', 'start' and 'duration' keys
         :rtype: [{'text': str, 'start': float, 'end': float}]
+        :param proxy: a dictionary mapping of http and https proxies to be used for the network requests
+        :rtype {'http': http_proxy, 'https': https_proxy}
```

Comment:
The docstring and README could also reference http://docs.python-requests.org/en/master/user/advanced/#proxies. As this describes the possible inputs in more detail.